forms-flow-web/src/components/Form/List.scss

```diff
@@ -12,17 +12,18 @@
 }
 
 .published-forms-label{
-    color: #52C41A;
-    background: #EAFBE7 0% 0% no-repeat padding-box;
-    border: 1px solid #52C41A;
+    color: var(--published-form-label);
+    // #52C41A;
+    background: var(--published-background) 0% 0% no-repeat padding-box;
+    border: 1px solid var(--published-form-label);
     border-radius: 7px;
 }
 
 .unpublished-forms-label{
-    background: #E6F7FF 0% 0% no-repeat padding-box;
+    background: var(--unpublished-background) 0% 0% no-repeat padding-box;
     border-radius: 7px;
-    border: 1px solid #91D5FF;
-    color: #1890FF;
+    border: 1px solid var(--unpublished-label);
+    color: var(--unpublished-label);
```

Comment:
border & color have different color codes but the same variable name(unpublished-label) is used

Reply:
yes. but here border is solid .

Reply:
The label and border are aligned with the same color.